test/e2e/scheduling/priorities.go

```diff
@@ -527,6 +528,10 @@ func createBalancedPodForNodes(f *framework.Framework, cs clientset.Interface, n
 		}
 	}
 
+	errChan := make(chan error, 1)
```

Comment:
if more than one creation fails, the routines will be blocked before they can do `wg.Done()`.

Reply:
Should `size` be `len(nodes)` then?

Reply:
yes and select handling this channel should also summarize those errors

Reply:
Sounds good, I updated that and aggregated the errors from the error channel. Though, I am not sure how exactly the `wg.Wait()` works, should I have some different handling for that so it doesn't block the `select` statement or will the errors just get through once all the routines have finished?

Reply:
select will only handle single value from chan.

I propose sth like:
```
close(errChan)
for err := range errChan {
     ...
}
```

Reply:
@jkaniuk thanks, updated with what you said. please let me know if this looks right

Reply:
LGTM modulo one comment